aplib.net-demo/Assets/Scripts/Entities/Player/Movement.cs

```diff
@@ -109,6 +120,13 @@ private void MovePlayer()
 
         if (isWallColliding) HandleWallCollision(wallNormal);
         LimitVelocity(isOnSlope);
+
+        // Lastly, play a footstep sound when the player is grounded, at certain intervals
+        if (_isGrounded && _rigidbody.velocity.magnitude > 0.25f && _timer.Countdown <= 0)
```

Comment:
The Timer class should have a method `Passed` or `TimeUp` to check this. This reads more intuitively than `.Counterdown <= 0`.